packages/rpc-core/src/params-patcher-options-object-position-config.ts

```diff
@@ -0,0 +1,51 @@
+export const OPTIONS_OBJECT_POSITION_BY_METHOD: Record<string, number> = {
```

Comment:
I need your help going over this with a fine-toothed comb.

1. Compare it to the docs to make sure I got the options argument position right.
2. Make sure I didn't miss one (check both one-shot and subscriptions)